blog/_posts/2017-09-11-ifPairElseUnknown-final-month.md

```diff
@@ -0,0 +1,119 @@
+* Developed our skills in programming; Ruby, Rails and Javascript.
+* Gained confidence in our ability to write clean and readable code.
+* Gained an understanding of TDD by writing lots of tests.
+* Networked with wonderful and supportive developers.
+* Connected with the amazing [RGSoC Community](https://railsgirlssummerofcode.org/) and [Ruby Australia](https://ruby.org.au/).
+* Shared our experiences on our [Twitter](https://twitter.com/IfPairElse) and through our [Daily logs.](https://ifpairelseunknown.github.io/)
+* Learnt about many wonderful Developer Tools.
+* Ate lots of chocolate.
+
+![Miracle](/img/blog/2017/ifPairElse_celebrate.gif)
+<div class="image-credits">Sourced from <a href="https://giphy.com">giphy.com</a></div>
+
+#### __What's next?__
+
+Our plans/goals for after RGSoC:
+
+We plan to develop on our existing programming skills, with the aim to become full time developers. We would like to share our experience participating in RGSoC with others, in the hope that this will encourage more diversity within the industry. We look forward to attending the [Ruby Australia Conference](http://www.rubyconf.org.au/2018) in 2018.
+
+#### __Thank You!__
+
+Our summer (winter) of code is coming to an end, but it has been a great pleasure working with our amazing team of coaches. They have all provided us with a wealth of knowledge, which we can take away and implement as we continue to learn. __Thank You__ to our wonderful coaches; __[Adel Smee](https://twitter.com/adelsmee), [Adam Rice](https://twitter.com/HashNotAdam), [Simon Hilderbrandt](https://au.linkedin.com/in/simonhildebrandt)__ and __[Tim Moore](https://twitter.com/tmoore)__
+
+A __Thank You__ to __[Luís Ferreira](https://twitter.com/zamith)__ for providing awesome support remotely.
+
+A __Thank You__ to __[Zendesk](https://www.zendesk.com/)__ for providing a welcoming and supportive environment for us to work on our project.
+
+__Thank You__ to our Project mentor __[Julia Nguyen](https://twitter.com/fleurchild)__ it has been a pleasure working on your project and we look forward to continuing our work on [if-me.org](https://www.if-me.org/). We also want to thank Julia and Bee, for our personalised RGSoC stickers and the wonderful if-me community for their support.
+
+![ifme stickers](/img/blog/2017/ifPair_ifme_stickers.jpg)
+<div align="center" div class="image-credits">Photo taken by Jenny and Sophie</div>
+
+__Thank You__ to our RGSoC Supervisor __Vi Nguyen__, it has been a pleasure getting to know you.
+
+Lastly, we would like to say a __BIG Thank You__ to the organisers of [Rails Girls Summer of Code](https://railsgirlssummerofcode.org/) and all the sponsors for making this experience possible. This has been such a wonderful experience, we have enjoyed the opportunity to contribute back to the community in a positive way. This is an experience we will never forget!
+
+### __You can find here__
```

Comment:
Hi, Thanks for sending a PR before scheduled date. The blog post looks good to me. :)

Only comment I've is I think you meant 'You can find **us** here' in the above line. Can you please fix it?